### ISSUE TITLE
Nix error

### ISSUE THREAD

**Issue by RamiroPastor** [sol/hpack#282]
```
Configuring hpack-0.18.1...
Setup: At least the following dependencies are missing:
aeson >=0.11
builder for '/nix/store/r1gsp5a6xcsch1hkxy4g826dz99c6cvd-hpack-0.18.1.drv' failed with exit code 1
```

Hi, i get this with `nix-build` with a very small miso app. I hope this is the place to solve this because i don't really know where to ask

**Comment by quasicomputational** [comment]
hpack doesn't know anything about nix or miso, so I'm afraid you'll probably need to look elsewhere; your best bet will probably be from the nix or Haskell-nix people.

I'll note that 0.18.1 was released a while ago and that hpack is on 0.28.1 now, so it's suspicious that you're building an old version.

**Comment by RamiroPastor** [comment]
Ok thx
